exercises/sum-of-multiples/sum_of_multiples_test.py

```diff
@@ -54,6 +54,16 @@ def test_multiples_of_1_up_to_100(self):
     def test_multiples_of_an_empty_list_up_to_10000(self):
         self.assertEqual(sum_of_multiples(10000, []), 0)
 
+    def test_multiples_of_0_up_to_1(self):
+        self.assertEqual(sum_of_multiples(1, [0]), 0)
+
+    def test_multiples_of_0_or_3_up_to_4(self):
+        self.assertEqual(sum_of_multiples(4, [0, 3]), 3)
+
+    def test_multiples_of_2_3_5_7_or_11_up_to_10000(self):
+        self.assertEqual(sum_of_multiples(10000,
+                         [2, 3, 5, 7, 11]), 39614537)
+
```

Comment:
In https://github.com/exercism/problem-specifications/commit/8f8975109653dc798eec74f0f239ddc25af71892#diff-fbfc29c1efc68e4b7c62f38d701c6572, test case names were changed to reflect the significance of each case. Can you please update the test names here as well?

Reply:
No problem.